tests/city_temporal_tiler_test_data/test_config_2009.yml

```diff
@@ -1,5 +0,0 @@
-PG_HOST: localhost
```

Comment:
After those changes do we still need those configuration files ?
Because on a quick glance, it seems that Test_Tile::setUpClass(cls) doesn't use them...

Reply:
The .yml configuration files aren't required anymore, that's why all of them are deleted in this PR

Reply:
Damned. My bad. Resolving.